lib/src/connector/ischool_plus_connector.dart

```diff
@@ -34,6 +34,12 @@ class ISchoolPlusConnector {
   static const String _getCourseName = "${_iSchoolPlusUrl}learn/mooc_sysbar.php";
   static const _ssoLoginUrl = "${NTUTConnector.host}ssoIndex.do";
 
+  // The Authorization Step of ISchool (2023-10-21)
+  // 0. POST https://app.ntut.edu.tw/ssoIndex.do
+  // 1. POST https://app.ntut.edu.tw/oauth2Server.do
```

Comment:
[Question]
Is there anything can guarantee that the `jumpUrl` will always be `oauth2Server.do` ?
If not, should we add some description here ?

Reply:
Fix on c82b1c8 and 6cfe60c